Plugins/BTCPayServer.RockstarDev.Plugins.Payroll/BTCPayServer.RockstarDev.Plugins.Payroll.csproj

```diff
@@ -21,7 +21,7 @@
 	</PropertyGroup>
 
 	<!-- This will make sure that referencing BTCPayServer doesn't put any artifact in the published directory -->
-	<ItemDefinitionGroup>
+	<ItemDefinitionGroup Condition="'$(Configuration)' == 'Release'">
```

Comment:
This line breaks debugging on localhost.

![image](https://github.com/user-attachments/assets/f50910fa-2f40-4d67-8fa0-caad1b36d90a)

I'll address it

Reply:
41075a4b7bded90ddbaf64ee1a5210de0bf14742